integration/single-node-slug-deploy/check.go

```diff
@@ -288,6 +291,17 @@ func verifyHelloRunning() error {
 		preparerT.Run()
 		return fmt.Errorf("Couldn't start hello after 15 seconds: \n\n hello tail: \n%s\n\n preparer tail: \n%s", helloTail.String(), preparerTail.String())
 	case <-helloPidAppeared:
+		log.Println("Hello PID appeared, letting start up and then checking for responsiveness")
+		<-time.After(10 * time.Second)
+		resp, err := http.Get("http://localhost:43770/_status")
```

Comment:
If you rebase on master you could use https://github.com/square/p2/blob/master/integration/single-node-slug-deploy/check.go#L137 as `checkStatus(43770, "hello")` that I added in PR #180